ansible/config_sonic_basedon_testbed.yml

```diff
@@ -517,6 +517,10 @@
         shell: config save -y
         when: save is defined and save|bool == true
 
+      - name: copy config db
+        become: True
+        shell: cp /etc/sonic/config_db.json /etc/sonic/config_db.json.bak
```

Comment:
Suggestions:
1. Update the description to "Backup the working golden config db". "Copy config db" is describing the action which is obvious by reading the code. "Backup the working golden config db" describes the purpose which would be more helpful for people reading the code.
2. Filename `config_db.json.bak` is also used by other scripts that need to backup config_db for temporary config change. Those scripts would overwrite your backup. Suggest to use a more unique filename like `/etc/sonic/config_db.json.golden.bak`.

Reply:
done